webpack.config.js

```diff
@@ -124,27 +123,13 @@ const reactNativeConfig = {
  *
  *   1. This package is designed for Node only and hence requires polyfills of Node libraries (e.g. `stream`, `path`) — webpack takes care of this for us.
  *   2. The package is not compatible with RequireJS and hence we don’t have any easy way to directly load it in our tests — the webpack bundle exposes it as a global named MochaJUnitReporter.
- *
- * - Node: The library uses optional chaining syntax, which is not supported by Node 12.
  */
 function createMochaJUnitReporterConfigs() {
```

Comment:
This function is still written as if supporting multiple configs. I think that the documentation comment, function name, and return value can all be simplified now that this is only for browser.

Reply:
True, simplified